helm/templates/api/api-configmap.yaml

```diff
@@ -297,7 +297,10 @@ data:
     applications:
       {{- .Values.applications | toYaml | nindent 6 }}
     {{- end }}
-
+    {{- if .Values.api.domains.reporters }}
+    reporters:
+      {{- .Values.api.domains.reporters | toYaml | nindent 6 }}
+    {{- end }}
```

Comment:
**issue**: Should be under domains section in the same way as identities. so the indent should be 8. In addition can you add unit test for this config ?